dev/PushNotifications/PushNotificationsLongRunningTask/platform.h

```diff
@@ -23,13 +23,18 @@ struct __declspec(uuid(PUSHNOTIFICATIONS_IMPL_CLSID_STRING)) NotificationsLongRu
 
     STDMETHOD(UnregisterForegroundActivator)(_In_ PCWSTR processName) noexcept;
 
+    STDMETHOD(AddToastRegistrationMapping)(_In_ PCWSTR processName, _In_ PCWSTR appId) noexcept;
+
+    STDMETHOD(RemoveToastRegistrationMapping)(_In_ PCWSTR processName) noexcept;
 private:
 
     std::map<std::wstring, std::wstring> GetFullTrustApps();
     const std::wstring GetAppIdentifier(std::wstring const& processName);
+    const std::wstring BuildAppIdentifier(std::wstring const& processName);
```

Comment:
Where is this being used?

Reply:
This being used to replace GetAppIdentifier. GetAppIdentifier is only a getter now and BuildAppIdentifier will build an appId if it hasn't been built yet.